analysis/subset_data/SubsetBam.wdl

```diff
@@ -0,0 +1,169 @@
+workflow SubsetBam {
+  # Bam to subset
+  File aligned_gold_bam
+
+  # How many cells to grab from the largest and smallest end
+  Int num_large_cells
+  Int num_small_cells
+
+  # How many reads a cell must have before it can be selected
+  Int? min_reads_per_cell
```

Comment:
It's actually a floor for any cell barcode selected by the tool.  I added this because I saw a lot of 1 read cell barcodes and this was the naive approach to give the ability to get rid of those cell barcodes.  I could add values for both the floor of large and small cell barcodes, that may make more sense and be more useful.  If you or @jishuxu have reasonable defaults that would be great as well. I will change this variable name to cell barcode as well

Reply:
Thanks for the explanation. I think separate floors would be a good idea. It's also possible that we would just want to take a much larger number of `small_cell_barcodes` than we were previously.